packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx

```diff
@@ -124,263 +124,260 @@ export const WebPropertyDetailPage = (): JSX.Element => {
+                <TabTitleIcon>
+                  <PackageIcon />
+                </TabTitleIcon>
+                <TabTitleText>SPAs</TabTitleText>{' '}
+              </>
+            }
+            aria-label="SPA listing"
+          >
+            {!spaProperties.isLoading && isSpaPropertyListEmpty ? (
+              <EmptyInfo propertyIdentifier={propertyIdentifier} />
+            ) : (
+              <>
+                <div className="pf-u-w-50 pf-u-mb-lg pf-u-mt-md">
+                  <Split hasGutter className="pf-u-mb-md">
                     <SearchInput
                       placeholder="Search by name"
                       value={searchTerm}
                       onChange={(value) => setSearchTerm(value?.toLowerCase())}
                       onClear={() => setSearchTerm('')}
                     />
-                  </SplitItem>
-                  <SplitItem isFilled />
-                  <SplitItem isFilled />
-                  <SplitItem>
-                    <Select
-                      variant={SelectVariant.single}
-                      aria-label="filter Input"
-                      value="Select Enviroment"
-                      onToggle={setIsFilterOpen.toggle}
-                      onSelect={(e, value) => {
-                        if (value === 'Select Enviroment') {
-                          setFilterByEnv('' as string);
-                        } else {
-                          setFilterByEnv(value as string);
-                        }
-                        setIsFilterOpen.off();
+                    <SplitItem isFilled />
+                    <SplitItem isFilled />
+                    <SplitItem>
+                      <Select
+                        variant={SelectVariant.single}
+                        aria-label="filter Input"
+                        value="Select Enviroment"
+                        onToggle={setIsFilterOpen.toggle}
+                        onSelect={(e, value) => {
+                          if (value === 'Select Enviroment') {
+                            setFilterByEnv('' as string);
+                          } else {
+                            setFilterByEnv(value as string);
+                          }
+                          setIsFilterOpen.off();
+                        }}
+                        selections="Select Enviroment" // To be kept as Select
+                        isOpen={isFilterOpen}
+                        aria-labelledby="filter"
+                      >
+                        {webPropertiesKeys.map((envName, index) => (
+                          <SelectOption key={`${envName} + ${index + 1}`} value={envName} />
+                        ))}
+                      </Select>
+                    </SplitItem>
+                  </Split>
+                  {filterByEnv === 'Select Enviroment' || filterByEnv === '' ? (
+                    <p />
+                  ) : (
+                    <div
+                      style={{
```

Comment:
@kotwani2883 noticed you added this part in previous commit. 

@kotwani2883 @ayushsatyam146 Kindly go thorugh pfe patternfly and next time try to use that. Let's avoid putting style like this as much as possible.

Reply:
Noted @akhilmhdh will change and raise the PR